<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerPanel</class>
 <widget class="QMainWindow" name="ServerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <widget class="QTextBrowser" name="LogsBrowser">
        <property name="placeholderText">
         <string>##################Logs are empty#########################</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="typeCommandEdit">
          <property name="placeholderText">
           <string>Type your message here:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SendToEveryoneButton">
          <property name="text">
           <string>Send to everyone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SendPrivateButton">
          <property name="text">
           <string>Send private</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <item>
       <widget class="QLineEdit" name="TypeUserNameEdit">
        <property name="placeholderText">
         <string>Type user name here:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QPushButton" name="searchOptionsButton">
          <property name="text">
           <string>Set search options</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="UsersList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_available_commands">
   <property name="text">
    <string>View available commands</string>
   </property>
  </action>
  <action name="actionView_Blacklist">
   <property name="text">
    <string>View blacklist</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
